Resources/public/js/views/ez-usermenuitemfireeventview.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuitemfireeventview', function (Y) {
+    "use strict";
+    /**
+     * Provides the user menu item fire event view
+     *
+     * @module ez-usermenuitemfire event view
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The user menu item fire event view. 
+     *
+     * @namespace eZ
+     * @class UserMenuItemFireEventView
+     * @constructor
+     * @extends Y.eZ.UserMenuItemView
```

Comment:
I think the `Y.` should be omitted so that yuidoc finds the reference and generates a link.